.github/workflows/test.yml

```diff
@@ -157,7 +162,7 @@ jobs:
     runs-on: ubuntu-latest
     name: GreenFrame
     needs: [e-commerce]
-    if: success() && github.ref == 'refs/heads/master'
+    if: success() && github.ref == 'refs/heads/master' && github.ref_type != 'tag'
```

Comment:
**thought**: on the contrary, we could run the Greenframe analysis only when a new tag/version is released, as opposed to any commit on `master` which might be a little too often. Wdyt?

Reply:
@fzaninotto any opinion?

Reply:
I agree that we should only test greenframe when releasing a new version

Reply:
Done